plugins/woocommerce/tests/e2e-pw/tests/merchant/product-import-csv.spec.js

```diff
@@ -95,6 +95,19 @@ const errorMessage =
 test.describe( 'Import Products from a CSV file', () => {
 	test.use( { storageState: process.env.ADMINSTATE } );
 
+	test.beforeAll( async ( { baseURL } ) => {
+		const api = new wcApi( {
+			url: baseURL,
+			consumerKey: process.env.CONSUMER_KEY,
+			consumerSecret: process.env.CONSUMER_SECRET,
+			version: 'wc/v3',
+		} );
+		// make sure the currency is USD
+		await api.put( 'settings/general/woocommerce_currency', {
+			value: 'USD',
+		} );
+	} );
+
```

Comment:
The solution in to perform an API call setting the currency to USD before the tests that were then failing.